### DISCUSSION TITLE
showing backup size

### DISCUSSION THREAD

**Discussion by benutzer74** [seedvault-app/seedvault#204] [Ideas]
EDIT: my concern was based on lack of knowledge how these backups work, I think it can be closed...

If Im not wrong, SV does not report the size of the backup prior performing the backup.
I think it would be useful if SV reports the needed space  before executing the backup.
Imagine you connect a usb thumbdrive for the backup and you have no notice it might not have enough space left......

**Comment by grote** [comment]
Indeed, the AOSP backup API doesn't inform us beforehand about the required space. We can tell it a quota and it will not backup apps above that quota. The real space requirements we only learn while performing the backup.
